src/test/java/com/u1/user/integrationtest/UserIntegrationTest.java

```diff
@@ -180,4 +183,25 @@ class createClass {
 
         }
     }
+
+    @Nested
+    class deleteClass {
```

Comment:
クラス名はアッパーキャメルケースがよいです。
```suggestion
    class DeleteClass {
```

Reply:
キャメルケースに修正致しました。

[7b5873b](https://github.com/u-1pena/assignment10/pull/15/commits/7b5873b5296d14f0b6e4a2d606fa55aa83a2e478)